contents/handbook/engineering/release-new-version.md

```diff
@@ -6,6 +6,10 @@ showTitle: true
 
 At the moment, we release a new version every month ([unless it makes sense not to!](/blog/we-ship-whenever)). This might change in the future.
 
+For consistency, releases happen on the last Monday of every month. Code freezes and break the release happen on the Wednesday before that. Each month there will be a different release owner in charge of the release, to be updated [here](https://calendar.google.com/calendar/embed?src=c_n8hc1iedb0k8gqhuiv83jolm50%40group.calendar.google.com&ctz=America%2FNew_York).
```

Comment:
I'd prefer we pick a specific date in the month. I never know when the last monday of the month is without checking a calendar, but "the 22nd every month" is pretty obvious. I think we can add a `*` that says if this date falls on a weekend we'll do it the monday after.

For the rotation, it makes sense to use pagerduty as we use it for all our other rotations.

Reply:
I think the issue @liyiy is trying to solve here is that if the day of the week we release in is ambiguous we don't know how all other pieces fit in. We don't want releases scheduled for Friday, or the weekend. Probably not Thursdays either. Also we would maybe benefit from having break the release be consistent - if it's 3 days before Day X and Day X is a Tuesday, break the release would fall on a Thursday, which we don't want.

We'll drop these in the calendar which will make it easy to know when it will happen, so I'm fine with this but don't feel strongly.

Reply:
I read that's what GitLab does but unless we're willing to release on weekends or Fridays/Thursdays and also code freeze/break the release on those days too, then there's no point in picking a specific day out of the month if we won't respect it. 

If we don't remember what the last Monday of every month is we can just check the calendar but that will always just be 1 rule to remember vs remembering that it's the "22nd of each month" but "unless that's a weekend or friday or thursday then it's a different day" then that's like 3 more rules you have to remember AND you still don't know which day it is anyway because no one knows which day the 22nd of each month falls on.. 

I do think that we should stick with the last Monday of each month because it's an easy rule to remember but if you feel very strongly about picking a specific date feel free to update this 😺

Reply:
Okay sounds good to me!